### DISCUSSION TITLE
Jekyll unable to use `--lsi` even with classifier-reborn installed in github action

### DISCUSSION THREAD

**Discussion by iwishiwasaneagle** [jekyll/jekyll#8932] [General]
I've been using github actions to automate my build and upload process for a while now. Recently however the builds started failing. The error messaged claimed classifier-reborn was not installed. The image below shows that it definietly is. Could someone provide some assistance please.

![image](https://user-images.githubusercontent.com/12383029/149029485-6dda4102-0450-44df-90da-2a27a55307c6.png)

**Comment by ashmaroli** [comment]
The clue is in the error message:
```
cannot load such file -- matrix
```
Ruby 3.1 stopped bundling `matrix` as part of the language distribution.
Therefore, you have add `gem "matrix"` to your Gemfile when building with Ruby 3.1 and above.

**Reply by iwishiwasaneagle** [reply]
Ah well that makes perfect sense. Thanks!
